CHANGELOG_DEV.md

```diff
@@ -1,3 +1,9 @@
+# 0.94.0 - 2022-08-18
+- Removed deprecated `--tls_db_sni` flag. Now is available only `--tls_database_sni`.
```

Comment:
```suggestion
- Removed deprecated `--tls_db_sni` flag. Now only `--tls_database_sni` is available.
```

Reply:
agree, will do it in next batch of commits to run CI once